ads/remixd.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS-IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function remixd(global, data) {
+  window._rmxd = {};
+  window._rmxd.url = data.url || window.context.sourceUrl;
+  const sriptVersion = data.version || '5';
+  const tagUrl = 'https://tags.remixd.com/player/v'+ sriptVersion +'/index.js?cb=' + Math.random();
+
+  document.write(
```

Comment:
Please use `loadScript` or `writeScript` from `3p.js` here

Reply:
The reason we are not using writeScript is that the script we need to write needs to have an id set. I saw that writeScript doesn't have an option to set the element id, do you have any other suggestion or can we keep the code like this?

Reply:
Yeah I guess not, so this is ok. Can you change it to use the passed in `global.document` though? Accessing document directly may do weird things based on different serving contexts.